kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/Utils.java

```diff
@@ -208,7 +208,7 @@ public static JdbcConnectionParams extractURLComponents(String uri, Properties i
       uri = uri.replace(urlPrefix, urlPrefix + authorityFromClientJdbcURL);
     }
     connParams.setSuppliedURLAuthority(authorityFromClientJdbcURL);
-    uri = uri.replace(authorityFromClientJdbcURL, dummyAuthorityString);
+    uri = uri.replaceFirst(authorityFromClientJdbcURL, dummyAuthorityString);
```

Comment:
Without this modification, UT can still pass, UT may not need `URLEncoder.encode`.

Reply:
done